src/Speckle.Sdk/Transports/ServerTransport.cs

```diff
@@ -28,20 +31,30 @@ public sealed class ServerTransport : IServerTransport
   /// <param name="timeoutSeconds"></param>
   /// <param name="blobStorageFolder">Defaults to <see cref="SpecklePathProvider.BlobStoragePath"/></param>
   /// <exception cref="ArgumentException"><paramref name="streamId"/> was not formatted as valid stream id</exception>
-  public ServerTransport(Account account, string streamId, int timeoutSeconds = 60, string? blobStorageFolder = null)
+  public ServerTransport(
+    ISpeckleHttp http,
+    ISdkActivityFactory activityFactory,
+    Account account,
+    string streamId,
+    int timeoutSeconds = 60,
+    string? blobStorageFolder = null
+  )
```

Comment:
@JR-Morgan I can see how the DX of this may not be great for Automate or other scripts, we should chat about what would the main pain-points are and come up with some guideline at least.